src/sinks/util/http.rs

```diff
@@ -346,16 +346,22 @@ impl RetryLogic for HttpRetryLogic {
 #[serde(deny_unknown_fields, rename_all = "snake_case", tag = "strategy")]
 pub enum Auth {
     Basic { user: String, password: String },
+    Bearer { token: String },
 }
 
 impl Auth {
     pub fn apply<B>(&self, req: &mut Request<B>) {
+        use headers::HeaderMapExt;
+
         match &self {
             Auth::Basic { user, password } => {
-                use headers::HeaderMapExt;
                 let auth = headers::Authorization::basic(&user, &password);
                 req.headers_mut().typed_insert(auth);
             }
+            Auth::Bearer { token } => match headers::Authorization::bearer(&token) {
+                Ok(auth) => req.headers_mut().typed_insert(auth),
+                Err(error) => error!(message = "invalid bearer token", %token, %error),
```

Comment:
```suggestion
                Err(error) => error!(message = "invalid bearer token.", %token, %error),
```

Reply:
I'm not sure, but maybe it's worth not log token value? Because there can be sensitive information.

Reply:
Good point, we could remove that from both places.